spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'users#index', type: :feature do
+  before(:each) do
+    @users = [
+      @user1 = User.create(
+        name: 'Munish',
+        photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+        bio: 'Web developer from Malawi',
+        posts_counter: 0
+      ),
+      @user2 = User.create(
+        name: 'Halai',
+        photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+        bio: 'Software Engineer.',
```

Comment:
- [ ] Dear @hmunish, For the profile picture we need to make sure we `can see the profile picture for each user`, hence Please apply a different profile picture and check if both are visible. Currently even if one of the profile picture is not visible the test will pass, snd this is not the expected behaviour.